db/Pos/DB/Class.hs

```diff
@@ -183,13 +187,14 @@ gsMaxProposalSize = bvdMaxProposalSize <$> gsAdoptedBVData
 gsUnlockStakeEpoch :: MonadGState m => m EpochIndex
 gsUnlockStakeEpoch = bvdUnlockStakeEpoch <$> gsAdoptedBVData
 
+-- | Checks if provided epoch is in the bootstrap era (pure version)
+isBootstrapEraPure :: EpochIndex -> EpochIndex -> Bool
+isBootstrapEraPure unlockEpoch curEpoch =
+    not isDevelopment && isBootstrapEra unlockEpoch curEpoch
```

Comment:
It's absolutely bullshit, because there is `isBootstrapEra` (and `not isDevelopment` should be ~removed~ there), but I am going to do CSL-1292 today, so I will fix it later.